kanidmd/src/lib/constants/ldap.rs

```diff
@@ -0,0 +1,192 @@
+//! Constants for the LDAP server
+
```

Comment:
This could be in https://github.com/kanidm/ldap3_server potentially?

Reply:
It sure could, I didn't realise it was its own crate :)

Reply:
For now we can do it here until it works, then we can add this to ldap3_server later :)